lab_6_pipeline/pipeline.py

```diff
@@ -144,17 +241,63 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._stemmer = Mystem()
+        mapping_path = Path(__file__).parent / 'data' / 'mystem_tags_mapping.json'
+        self._converter = MystemTagConverter(mapping_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = []
+        result = self._stemmer.analyze(re.sub(r'\W+', ' ', text))
+        token_count = 0
+        for sentence_position, sentence in enumerate(split_by_sentence(text)):
+            conllu_tokens = []
+            for token_position, token in enumerate(re.findall(r'\w+', sentence), start=1):
+                conllu_token = ConlluToken(token)
+                if not result[token_count]['text'].isalnum():
+                    token_count += 1
```

Comment:
why do you increase it here? and even allow it to be processed further. provide examples

Reply:
fixed